salt/sdb/vault.py

```diff
@@ -0,0 +1,154 @@
+# -*- coding: utf-8 -*-
+'''
+Vault Database Module
+
+:maintainer:    SaltStack
+:maturity:      New
+:depends:       vault (Hashicorp)
+:platform:      all
+
+.. versionadded:: Carbon
+
+This module allows access to Hashicorp Vault using an ``sdb://`` URI.
+
+Like all sdb modules, the vault module requires a configuration profile to
+be configured in either the minion or master configuration file. This profile
+requires very little. In the example:
+
+.. code-block:: yaml
+
+    myvault:
+      driver: vault
+      vault.host: 127.0.0.1
+      vault.port: 8200
+      vault.scheme: http  # Default is https
+      vault.token: 012356789abcdef  # Optional
+
+The ``driver`` refers to the ``vault`` module, ``vault.host`` refers to the host
+that is hosting vault and ``vault.port`` refers to the port on that host. A
+vault token is also required. It may be set statically, as above, or as an
```

Comment:
This says the vault token is "also required" but there is a comment in code-block that says it's optional. Can you clarify this in the docs here?